<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdDialog</class>
 <widget class="QDialog" name="ThirdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(250, 255, 155);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    display: block;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 18px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: none;
        color: #666;
        position: absolute;
         
        -webkit-mask-image: -webkit-gradient(linear, left top, left bottom, from(rgba(0,0,0,1)), color-stop(50%, rgba(0,0,0,.5)), to(rgba(0,0,0,1)));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Recupereaza-ti parola:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: none;
        color: #666;
        position: absolute;
         
        -webkit-mask-image: -webkit-gradient(linear, left top, left bottom, from(rgba(0,0,0,1)), color-stop(50%, rgba(0,0,0,.5)), to(rgba(0,0,0,1)));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Intrebare:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>100</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: none;
        color: #666;
        position: absolute;
         
        -webkit-mask-image: -webkit-gradient(linear, left top, left bottom, from(rgba(0,0,0,1)), color-stop(50%, rgba(0,0,0,.5)), to(rgba(0,0,0,1)));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Raspuns:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
font-family: 'Segoe UI', 'Open Sans', Arial, sans-serif;
    display: block;
    color: rgb(255, 255, 255);
    text-decoration: none;
    text-align: center;
    padding: 10px;
    margin: 15px 0px 0px 15px;
    font-size: 18px;
    text-transform: uppercase;
    background: #ED5E2F;
    color: #FFF;
    border: 0px none;
    outline: 0px none;
}

.QPushButton:hover {
  background: #3498db;
  background-image: -webkit-linear-gradient(top, #3498db, #3498db);
  background-image: -moz-linear-gradient(top, #3498db, #3498db);
  background-image: -ms-linear-gradient(top, #3498db, #3498db);
  background-image: -o-linear-gradient(top, #3498db, #3498db);
  background-image: linear-gradient(to bottom, #3498db, #3498db);
  text-decoration: none;
}

.QPushButton:pressed {
    background: #F6A953;
}</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ThirdDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ThirdDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
